events/2020-10-10.interspeciesio-ml-workshop.md

```diff
@@ -1,25 +1,109 @@
-# October 10, 2020 - Earth Species Project Machine Learning & Bioacoustics Workshop hosted by Interspecies.IO
+# October 10, 2020 - Earth Species Project Machine Learning & Bioacoustics Workshop
 
-**TODO:** Copy describing the meeting
-- Time/Date (with link to https://savvytime.com/converter/pst-to-mst-est-cest or some such calculator)
-- less info in the title and more here?
-- Attendees? (greater consideration given the number of people we had in attendance)
+## hosted by Interspecies.IO
 
-## Watch
+It was a tremendous honor to present to the [Interspecies.io](https://interspecies.io/) community and share our latest work. The workshop was hosted by Interspecies.IO, a multidisciplinary group founded by Peter Gabriel (musician and activist), Dr. Vint Cerf (Vice President at Google and Co-designer of the Internet), Dr. Diana Reiss (Professor at Hunter College, a cognitive psychologist, and marine mammal scientist), and Dr. Neil Gershenfeld (head of MIT’s Center for Bits and Atoms).
 
+## Attendees: 
+
+Nearly 100 biologists, computational linguists, information theorists, musicians, and technologists. Continue the conversation with us [here](https://github.com/earthspecies/project/discussions/35). 
```

Comment:
- Do we even need this? Feels like tooting our own horn a bit. Not super opinionated so I defer to you + @aza.
- If we keep, I'm not a huge fan of the term technologists. My suggestion included in this comment.

```suggestion
Nearly 100 biologists, computational linguists, computer scientists and engineers, information theorists, musicians, and artists. Continue the conversation with us [here](https://github.com/earthspecies/project/discussions/35). 
```

Reply:
I didn't know how we would best refer to folks like Vint C., Neil and ML researchers on the call. Technologists was the best catch-all I could come up with but I prefer your language.

Reply:
I do think it is nice to call out the audience. It's more of a tooting Interspecies.io horn in a way ;)